volcli/volcli.go

```diff
@@ -125,7 +135,7 @@ func TenantList(ctx *cli.Context) {
 // VolumeCreate creates a new volume with a JSON specification to store its
 // information.
 func VolumeCreate(ctx *cli.Context) {
-	if len(ctx.Args()) != 2 {
+	if len(ctx.Args()) != 1 {
 		errExit(ctx, fmt.Errorf("Invalid arguments"), true)
```

Comment:
Yeah, I think this is an artifact of me doing this too quickly initially. #108 was just created so this can be corrected.